mage_integrations/mage_integrations/sources/paystack/streams/base.py

```diff
@@ -0,0 +1,166 @@
+from datetime import datetime, timedelta
+from typing import Dict, List
+from dateutil.parser import parse
+from mage_integrations.sources.constants import REPLICATION_METHOD_INCREMENTAL
+from mage_integrations.sources.messages import write_schema
+from mage_integrations.sources.paystack.client import PaystackClient
+from mage_integrations.sources.state_utils import (
```

Comment:
Remove unused imports. Also if `state_utils` is not shared, you don't need to move that file.